<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QMainWindow" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>691</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox {
background-color:rgb(188, 202, 255);
border-radius:20px;
}

#play {
background-color:rgb(143, 160, 255);
border-radius:20px;
}

#newGame {
background-color:rgb(143, 160, 255);
border-radius:10px;
}

#leadersBoard {
background-color:rgb(143, 160, 255);
border-radius:10px;
}

#check {
background-color:rgb(143, 160, 255);
border-radius:10px;
}

#input {
background-color:rgb(227, 244, 255);
border-radius:10px;
color:rgb(85, 143, 255);
}

#label {
color:rgb(88, 58, 90);
}

#tableWidget {
background-color:rgb(227, 244, 255);
border-radius:10px;
}

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="mainLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>501</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>58</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bools and cows</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="play">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>370</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>52</pointsize>
      </font>
     </property>
     <property name="text">
      <string>play</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>131</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>26</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="check">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>161</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>check!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>591</width>
       <height>401</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>25</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="newGame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>520</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>new game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leadersBoard">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>520</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>leader board</string>
     </property>
    </widget>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>520</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BM Dohyeon</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>tableWidget</zorder>
    <zorder>mainLabel</zorder>
    <zorder>play</zorder>
    <zorder>input</zorder>
    <zorder>check</zorder>
    <zorder>label</zorder>
    <zorder>newGame</zorder>
    <zorder>leadersBoard</zorder>
    <zorder>status</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
